tests/unit/test_cli.py

```diff
@@ -71,14 +85,29 @@ async def test_entrypoint_real_run():
         "cou.cli.generate_plan"
     ), patch("cou.cli.apply_plan") as mock_apply_plan, patch("cou.cli.Analysis.create"):
         mock_parse_args.return_value = MagicMock()
-        mock_parse_args.return_value.dry_run = False
+        mock_parse_args.return_value.run = True
 
         result = await entrypoint()
 
         assert not result
         mock_apply_plan.assert_called_once()
 
 
+@pytest.mark.asyncio
```

Comment:
Can we add also test-case when we will test behavior running also sub sub step? You accidentally broke than in your previous commit and no tests were broken. Thanks

Reply:
I've added a third layer of sub_step to prove that works. Thanks